############### Localization parameters ############### 

## Measurement parameters
# Minimal height of the measurement to be considered valid
min_detection_height: 1.0  # meters above zero altitude
# Covariance parameters, applied to depth detection measurements
depth_detections:
  # Covariance of the detection measurement in the XY direction (in camera CS)
  xy_covariance_coeff: 0.3
  # Covariance coefficient of the detection measurement in the Z direction (in camera CS)
  # This coefficient is scaled by z*sqrt(z) to obtain the resulting covariance in Z direction.
  z_covariance_coeff: 0.3
  # Maximal mahalanobis distance between a measurement and a hypothesis for the measurement
  # to be considered for association to the hypothesis
  max_gating_distance: 2.5

# Covariance parameters, applied to rgb tracking measurements
rgb_trackings:
  # Covariance of the detection measurement in the XY direction (in camera CS)
  xy_covariance_coeff: 0.6
  # Covariance coefficient of the detection measurement in the Z direction (in camera CS)
  # This coefficient is scaled by z*sqrt(z) to obtain the resulting covariance in Z direction.
  z_covariance_coeff: 3.0
  # Maximal mahalanobis distance between a measurement and a hypothesis for the measurement
  # to be considered for association to the hypothesis
  max_gating_distance: 2.5

## Prediction parameters
# Process noise of the LKF position states
lkf_process_noise_pos: 1.0  # it is scaled by dt in the code, so should it be given scaled to seconds here
# Process noise of the LKF velocity states
lkf_process_noise_vel: 3.0  # it is scaled by dt in the code, so should it be given scaled to seconds here

## other hypothesis parameters
# If a hypothesis exceeds this uncertainty, it is kicked out of the pool of active LKFs.
max_hyp_uncertainty: 3.0
# Initial covariance of velocity states at LKF initialization
init_vel_cov: 4.0
# A LKF must have received at least this many corrections before it is considered as
# a candidate for publishing.
min_corrs_to_consider: 20
